application/src/main/kotlin/net/corda/v5/application/marshalling/JsonMarshallingService.kt

```diff
@@ -0,0 +1,32 @@
+package net.corda.v5.application.marshalling
+
+import net.corda.v5.base.annotations.DoNotImplement
+
+/**
+ * An optional service CorDapps and other services may use to marshall arbitrary content in and out of JSON format using standard/approved
+ * mappers.
+ */
+@DoNotImplement
+interface JsonMarshallingService : MarshallingService {
```

Comment:
The real issue with this service is that it provides the user no opportunity to adjust it, or register customer serializers. I hope there is a JIRA for that and we will need to create yet another marker interface/annotation for those custom serializers in same way as we have for Kryo and AMQP

Reply:
https://r3-cev.atlassian.net/browse/CORE-5190